### DISCUSSION TITLE
BFE后端服务如何支持https

### DISCUSSION THREAD

**Discussion by ZhongShawn** [bfenetworks/bfe#1144] [Q&A (问题和回答)] [answered]
目前BFE是不支持后端服务使用https协议的，请问下BFE后续是否有支持https的计划，或者有没有什么思路，可以支持后端服务使用https

**Comment by mileszhang2016** [comment, marked as answer]
BFE开源后续将支持基于HTTPS来访问后端服务。
计划在2024Q1内完成支持

**Comment by cc14514** [comment]
This feature has already been released in [v1.8.0.](https://github.com/bfenetworks/bfe/releases/tag/v1.8.0) Please upgrade to that version to use it.
